store/tikv/coprocessor.go

```diff
@@ -408,18 +410,14 @@ func (it *copIterator) run(ctx goctx.Context) {
 	// Start it.concurrency number of workers to handle cop requests.
 	for i := 0; i < it.concurrency; i++ {
 		copIteratorGP.Go(func() {
-			childCtx, cancel := goctx.WithCancel(ctx)
```

Comment:
Can we delete this directly without adding the `closed` field ?
AFAIK, the purpose of creating a new context before a new goroutine is to reduce the lock contention.
But maybe it's no problem here.